vulcan/lib/client/jsx/reducers/vulcan_reducer.ts

```diff
@@ -81,8 +82,8 @@ export default function VulcanReducer(state: VulcanState, action: VulcanAction):
 
       return {
         ...state,
-        session: action.session,
-        inputs: {...action.session.inputs},
+        session: {...state.session, ...action.session},
```

Comment:
One additional change to how `setSession` works, because it seems like parallel steps would overwrite user inputs if they were made during the "waiting" stage. Best example of this is color-by and differential gene expression clustering. The color-by UI widget appears before the differential gene expression cells finish running. If a user selected a color-by option while waiting, after the diffexp cells finished, their "status" update would wipe out the selected color-by option. This meant that clicking the Run button would do nothing, and the user would have to re-select a color-by option. Kind of confusing with no good way to provide feedback.

This change preserves any user inputs made during in the interim.